crates/shared/src/arguments.rs

```diff
@@ -103,9 +104,32 @@ pub struct OrderQuotingArguments {
 
 logging_args_with_default_filter!(
     LoggingArguments,
-    "warn,autopilot=debug,driver=debug,orderbook=debug,solver=debug,shared=debug,cow_amm=debug"
+    "info,autopilot=debug,driver=debug,orderbook=debug,solver=debug,shared=debug,cow_amm=debug"
 );
 
+#[derive(Debug, clap::Parser)]
+pub struct TracingArguments {
+    #[clap(long, env)]
+    pub tracing_collector_endpoint: Option<String>,
+    #[clap(long, env)]
+    pub tracing_level: Option<LevelFilter>,
+    #[clap(long, env, value_parser = humantime::parse_duration)]
+    pub tracing_exporter_timeout: Option<Duration>,
+}
+
+pub fn tracing_config(args: &TracingArguments, service_name: String) -> Option<TracingConfig> {
+    args.tracing_collector_endpoint.as_ref().map(|endpoint| {
```

Comment:
nit: using `let Some(endpoint) = &args.tracing_collector_endpoint else { return None; }` would make this easier to read IMO.